spec/rails_helper.rb

```diff
@@ -9,27 +9,30 @@ def jsonapi_response
   JSON.parse(response.body, symbolize_names: true)
 end
 
-# rails changed test case request params from positional to keywords in rails 5.1.7
+# rails changed controller params from positional to keywords in rails 5.1.7
 # 5.1.6>= rails >=5.0 consumes both ways.
 # https://apidock.com/rails/v5.1.7/ActionController/TestCase/Behavior/process
 # others keywords can be: session, body, flash, format, xhr, as.
 # We are using `as: json` in specs (rails 5+),
 # but rails 4 does not require providing that and consumes only params, session and flash.
+# Integration specs does not have session and flash but we don't require them so we will skip them.
+# Integration reqests can pass also headers, but we don't use it yet.
```

Comment:
Typo "reqests" - "requests"

Also I think it would sound better: "requests can also pass headers", instead of "can pass also headers"